models.md

```diff
@@ -0,0 +1,35 @@
+# Models definition
++ The store needs to be able to map type names to TS classes.
+
+### Proposition 1
+
+```typescript
+@JsonApiModel({
+  type: 'posts',
+  endpoint: '/posts',
+  baseUrl: 'http://api.example.com'
+})
+class Post {
+  @JsonApiAttribute()
+  title: string
+
+  @JsonApiAttribute()
+  content: string
+
+  @JsonApiAttribute({
+    key: 'published_at'
```

Comment:
I think this is why EmberData has an adapter :-(

They allow overriding of key transforms, so you have underscored keys, you can transform to camelCase or whatever.